lib/Makefile

```diff
@@ -24,11 +24,19 @@ FF_KNI=1
 #FF_IPFW=1
 
 ifeq ($(FF_DPDK),)
-    FF_DPDK=${TOPDIR}/dpdk/x86_64-native-linuxapp-gcc
+ifeq (${MACHINE_CPUARCH},aarch64)
+    FF_DPDK=${TOPDIR}/dpdk/build
+else
+    #FF_DPDK=${TOPDIR}/dpdk/x86_64-native-linuxapp-gcc
```

Comment:
Why is it removed?

Reply:
Sorry, that's a mistake. Should not remove FF_DPDK for x86.
How about:
ifeq ($(FF_DPDK),)
ifeq ($(shell uname -m),aarch64)
    FF_DPDK=${TOPDIR}/dpdk/build
else
    $(warning ${FF_DPDK})
    FF_DPDK=${TOPDIR}/dpdk/x86_64-native-linuxapp-gcc
endif
endif

ifdef RTE_SDK
ifeq (${shell uname -m},aarch64)
    FF_DPDK=${RTE_SDK}/build
else
    FF_DPDK=${RTE_SDK}/x86_64-native-linuxapp-gcc
endif
endif